### DISCUSSION TITLE
Explicitly support Relations to records within the same database

### DISCUSSION THREAD

**Discussion by therden** [RafaelGB/obsidian-db-folder#576] [Ideas]
Thank you @RafaelGB for this terrific plugin -- and most especially for the recent addition of Relations and Rollups!

At present (version 2.8.1), the Settings dialog for a Relations field excludes the current database from the list of databases to which it could linked.  

However, there are legitimate use cases for wanting to associate one record with another in the same database (for example: a supervisor field in an employee record linked to another record in the Employees database.)  

I've confirmed that when the note containing the database is manually edited outside of Obsidian, or if the database's own filepath within the vault is typed into the "Select another table to relate with" field of the dialog, both Relations and Rollups based on Relations fields do work as expected.

So:  I suggest including the current database in the list of databases to which a Relation can be established (and, I guess, changing the label within the dialog to something like "Select _the_ table to relate with".)

**Comment by RafaelGB** [comment]
hi @therden, for your use case you  already have the formulas. You can associate cells of the same row with the logic that you consider.

add the same ddbb into the relation list introduce some exceptions
